tron/core/action.py

```diff
@@ -31,6 +32,7 @@ class ActionCommandConfig:
     docker_parameters: set = field(default_factory=set)
     env: dict = field(default_factory=dict)
     secret_env: dict = field(default_factory=dict)
+    secret_volumes: List[ConfigSecretVolume] = field(default_factory=list)
```

Comment:
hmm, should we be consistent with `extra_volumes` and use a set here?

Reply:
I had troubles with that due to many tests trying to hash it when it's a set

Reply:
ah, I guess extra_volumes is something more hashable then? no worries then :)